lib/rules/length-zero-no-unit/__tests__/index.js

```diff
@@ -431,3 +494,15 @@ testRule(rule, {
 		},
 	],
 });
+
+testRule(rule, {
+	ruleName,
+	config: [true],
+	fix: true,
+
+	accept: [
+		{
+			code: '.a { grid-template-rows: 0fr 1fr; }',
```

Comment:
I think you can put this test where you added other tests, because stylelint configuration is the same.

Reply:
Ooops... Agree. Put it with other tests.
P.S. The last commit I did from Ubuntu and pre-commit hook worked as expected :confused: